test/Test/Sync.hs

```diff
@@ -0,0 +1,23 @@
+module Test.Sync
+       ( syncSpecs
+       ) where
+
+import IW.Core.Repo (RepoOwner (..), RepoName (..))
+import IW.Sync.Search (parseIssueUserData)
+import Test.Hspec (Spec, describe, it, shouldBe)
+
+import qualified GitHub  
+
+
+testGitHubIssueUrl :: GitHub.URL
+testGitHubIssueUrl = GitHub.URL "https://api.github.com/repos/owner123/repo123/issues/1"
+
+syncSpecs :: Spec
+syncSpecs = describe "GitHub sync correctness"
+    parseIssueUserDataSpec
+
+parseIssueUserDataSpec :: Spec
+parseIssueUserDataSpec = describe "parseIssueUserData" $
+    it "should return Just (RepoOwner owner123, RepoName repo123)" $ 
+        parseIssueUserData testGitHubIssueUrl 
+            `shouldBe` Just (RepoOwner "owner123", RepoName "repo123")
```

Comment:
Don't you mind to add a few more tests on wrong URLs, just to check that it doesn't parse wrong input?

Reply:
@vrom911 Good idea💡